migration_steps/prepare_source_data/counts_verification/sql/count_casrec_source.sql

```diff
@@ -151,11 +153,13 @@ UPDATE countverification.counts SET casrec_source =
     SELECT COUNT(*)
     FROM casrec_csv.pat
     WHERE casrec_csv.pat."Debt chase" != ''
+    AND casrec_csv.pat."Debt chase" != '0'
```

Comment:
Feel like this should be "Debt chase" = '1' like validation, as I'm getting hits for '0.0' using the above